larsim/IonizationScintillation/ISCalcNESTLAr.cxx

```diff
@@ -57,53 +57,33 @@ namespace larg4
         }
 
         int pdgcode = edep.PdgCode();
-
-        geo::Length_t startx = edep.StartX();
-        geo::Length_t starty = edep.StartY();
-        geo::Length_t startz = edep.StartZ();
-        geo::Length_t endx   = edep.EndX();
-        geo::Length_t endy   = edep.EndY();
-        geo::Length_t endz   = edep.EndZ();
-//        geo::Point_t  midpos = edep.MidPoint();
-
-//        double delta    = 1*CLHEP::mm;
-//        double R0           = 1.568*CLHEP::um;    //Mozumder 1995
-
+        
         double DokeBirks[3];
-//        double ThomasImel = 0.00;
 
-        double eField = EFieldAtStep(fDetProp->Efield(), edep);
+//        double eField = EFieldAtStep(fDetProp->Efield(), edep);
+        double eField = EFieldAtStep(fDetProp->Efield(), edep) * 1000;  //conver kV to 1000 V
```

Comment:
Could a CLHEP unit conversion be used here (I think other CLHEP conversions are used in this file)?
e.g. https://www.zeuthen.desy.de/ILC/geant4/clhep-2.0.4.3/SystemOfUnits_8h-source.html